frontend/app/routes/public/renew/$id/adult-child/confirm-phone.tsx

```diff
@@ -101,6 +101,8 @@ export async function action({ context: { appContainer, session }, params, reque
     })
     .transform((val) => ({
       isNewOrUpdatedPhoneNumber: val.isNewOrUpdatedPhoneNumber === AddOrUpdatePhoneOption.Yes,
+      phoneNumber: val.phoneNumber,
+      phoneNumberAlt: val.phoneNumberAlt,
```

Comment:
Spreading `val` should include them automatically.
```ts
.transform((val) => ({
  ...val,
  isNewOrUpdatedPhoneNumber: val.isNewOrUpdatedPhoneNumber === AddOrUpdatePhoneOption.Yes,
})
```

Reply:
Spreading `val` will fix the issue, no need to explicitly add phone number fields in `.transform()`.